geodata/urls.py

```diff
@@ -19,4 +26,5 @@
     path('api/geodata/near_point/extent', search.search_near_point_extent),
     path('api/geodata/raser/near_point/extent', search.search_near_point_extent_raster),
     path('api/geodata/geometry/near_point/extent', search.search_near_point_extent_geometry),
+    path('api/geodata/', include(router.urls)),
```

Comment:
I'm seeing this mounted at `/api/geodata/api/<model name>`.  Because swagger automatically hides common prefixes, it shows up in swaggeras `geodata/api/<model name>`.  If we mount everything under `geodata`, the swagger interface is grouped poorly.  As a consequence, I think we just want to mount this via `path('', include(router.urls))` and let the API list both the core and geodata models.  If we have models with the same names in each app, then my preference would be to override core, since it is actually our less important use case.

Reply:
Just to clarify, Dataset is in both, right now it overrides geodata's, so should I make it so that core's get overridden or should I just rename one of the models?

Reply:
For now, don't worry about which one wins (since I assume it has to do with import order).  Just make a new issue to revisit how to resolve this once the rest of this PR is done.